### DISCUSSION TITLE
Alpine-Zigbee2MQTT - symbolic link to config incorrect

### DISCUSSION THREAD

**Discussion by DorjeDorf** [tteck/Proxmox#2975] [Help Needed]
I tried it twice before reporting, but for whatever reason, I seem to always get the same error. 

Line 27 of the install script reads:
`ln -s /etc/zigbee2mqtt/ /root/.z2m
`
Which makes absolute sense and is ultimately what I used, however - in two cases for me - instead of making the symbolic link be /root/.z2m - what I would instead get is a symbolic called zigbee2mqtt within /root/.z2m.

This is what I was seeing:

```
alpine-zigbee2mqtt:~# ls -ltra
total 9
drwxr-xr-x    2 root     root          1024 May 13 12:21 .z2m
-rw-r--r--    1 root     root            29 May 13 12:21 .bashrc
drwxr-xr-x   20 root     root          1024 May 13 12:40 ..
-rw-------    1 root     root          3945 May 13 12:43 .viminfo
drwx------    3 root     root          1024 May 13 12:43 .
-rw-------    1 root     root           778 May 13 12:47 .ash_history
alpine-zigbee2mqtt:~# ls .z2m/
zigbee2mqtt
alpine-zigbee2mqtt:~# ls .z2m/ -ltra
total 2
lrwxrwxrwx    1 root     root            17 May 13 12:21 zigbee2mqtt -> /etc/zigbee2mqtt/
drwxr-xr-x    2 root     root          1024 May 13 12:21 .
drwx------    3 root     root          1024 May 13 12:43 ..
```

This caused zigbee2mqtt to crash on start each time. I had to manually delete the link and the .z2m folder and then use the exact same command to set up a proper link.

I'm using a fresh install of Proxmox 8.2.2 with very few LXCs (only at the setup stage). I am using Ceph and proxmox is operating in HA mode, but I doubt that should matter. 

Has anyone else seen this?

**Comment by nagi603** [comment]
Yeah, same was for me. Also the alpine version was (is?) 3.19 instead of 3.20 which meant I had to manually upgrade to get the last few months' worth of releases.
